src/views/content/content-page.tsx

```diff
@@ -4,13 +4,14 @@ import { flatten, toPairs } from 'lodash';
 import * as React from 'react';
 
 import { NamedSFC } from '../../common/react/named-sfc';
+import { GuidanceTag } from '../../content/guidance-tags';
 import { createMarkup, Markup, MarkupDeps } from './markup';
 
-export type HyperlinkDefinition = { href: string; text: string };
+export type HyperlinkDefinition = { href: string; text: string; tags?: GuidanceTag[] };
```

Comment:
It seems to me that altering the `HyperlinkDefinition` it's a little confusing. In general, I'd say, a hyperlink is just the text and the href (maybe just the href) but nothing more. It feels like we should have a new type where we have the `tags` (probably not optional).

Looking at the changes I found `getGuidanceTagsFromGuidanceLinks`, which suggest `GuidanceLink` for the new type... something like this maybe?
```typescript
type GuidanceLink = HyperlinkDefinition & { tags: GuidanceTag[] };
```

Reply:
Updated!